src/modules/nav/nav.component.html

```diff
@@ -2,6 +2,9 @@
   <ol class="stache-nav-list">
     <li class="stache-nav-list-item" *ngFor="let route of routes">
       <a class="stache-nav-anchor" (click)="navigate(route)" [ngClass]="{'active': isActive(route)}">{{route.name}}</a>
+      <div class="stache-nav-child-list">
```

Comment:
This DIV isn't really needed. We can target `.stache-nav .stache-nav` in the CSS.